index.html

```diff
@@ -342,18 +342,24 @@
       var mutationCount = 0;
 
       function translateTextNode(node, info) {
-        if(langify.settings.observeCustomContents === false || !node.textContent || node.textContent.trim().length === 0) {
+        if (langify.settings.observeCustomContents === false || Object.keys(customContents_text).length === 0) {
           return;
         }
-        var src = node.textContent.trim().replace(/(\r\n|\n|\r)/gim,"").replace(/\s+/g," ").toLowerCase();
-        if(customContents_text[src] && node.textContent !== customContents_text[src]) {
-          var newContent = node.textContent.replace(node.textContent.trim(), customContents_text[src]);
-          if (newContent != node.textContent) {
+              
+        var textContent = node.textContent || '';
+        var trimmedTextContent = textContent.trim();
```

Comment:
Both `textContent` and the trimmed version of it were used multiple times so we can cache them in the variable.